lang/en.yml

```diff
@@ -204,7 +204,8 @@ en:
     Editors: 'Editors Groups'
     HASBROKENLINKS: 'This page has broken links.'
     HTMLEDITORTITLE: Content
-    INHERIT: 'Inherit from parent page'
+    INHERITPARENT: 'Inherit from parent page'
```

Comment:
I'm wondering, if we would be better to keep `INHERIT` here to avoid breaking every single translation we have for the existing key?

Reply:
You're right, I thought I'd done a search and there wasn't any other translations for that label but I must've searched wrong because there are, so I've reverted it.